discovery-provider/alembic/versions/ac0c6f94dd8d_partial_created_at_idx.py

```diff
@@ -0,0 +1,85 @@
+"""partial_created_at_idx
+
+Revision ID: ac0c6f94dd8d
+Revises: df4709b4391b
+Create Date: 2022-03-03 21:11:42.415590
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "ac0c6f94dd8d"
+down_revision = "df4709b4391b"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+    query = """
+
+    -- make track_owner_id_idx a partial index
+    DROP INDEX track_owner_id_idx;
+    CREATE INDEX track_owner_id_idx ON tracks (owner_id) WHERE is_current and not is_delete;
+
+    -- add index for track_created_at
+    CREATE INDEX track_created_at_idx ON tracks (created_at) WHERE is_current and not is_delete;
```

Comment:
We also try to enforce that all our migrations are idempotent, so this should be `CREATE INDEX IF NOT EXISTS` thru-out.

https://github.com/AudiusProject/audius-protocol/blob/47ae25d6f80751f595c603aaee7d031ace953b84/discovery-provider/integration_tests/test_migration_idempotency.py#L31